en/messages/README.md

```diff
@@ -1,13 +1,15 @@
 <!-- THIS FILE IS AUTO-GENERATED (DO NOT UPDATE GITBOOK): https://github.com/mavlink/mavlink/blob/master/doc/mavlink_gitbook.py -->
 # Message Definitions
 
-MAVLink messages are defined in XML files in the [mavlink/message definitions](https://github.com/mavlink/mavlink/blob/master/message_definitions/) folder. The messages that are common to all systems are defined in [common.xml](https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/common.xml) (only messages contained in this file are considered standard messages). MAVLink protocol-specific messages and vendor-specific messages are stored in separate XML files.
+MAVLink messages are defined in XML files in the [mavlink/message definitions](https://github.com/mavlink/mavlink/blob/master/message_definitions/) folder. The messages that are common to all systems are defined in [common.xml](https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/common.xml) (only messages contained in this file are considered standard messages).
+
+MAVLink protocol-specific messages and vendor-specific messages are stored in separate XML files. These are referred to as *dialects*.

```

Comment:
Pleas add a subtitle and anchor here: `Vender specific extensions (dialects)`
That way the links you added above can link directly to this list

Reply:
I meant to add this before line 6

Reply:
Great idea. Did this in https://github.com/mavlink/mavlink/pull/902 - but I'll just pull the generated content in here for now.